test/parallel/test-tls-max-send-fragment.js

```diff
@@ -33,10 +33,39 @@ const buf = Buffer.allocUnsafe(10000);
 let received = 0;
 const maxChunk = 768;
 
+const invalidArgumentError = {
+  name: 'TypeError',
+  code: 'ERR_INVALID_ARG_TYPE'
+};
+
+const rangeError = {
+  name: 'RangeError',
+  code: 'ERR_OUT_OF_RANGE'
+};
+
+const typeError = {
+  name: 'TypeError'
+};
+
 const server = tls.createServer({
   key: fixtures.readKey('agent1-key.pem'),
   cert: fixtures.readKey('agent1-cert.pem')
 }, function(c) {
+
+  // No size is passed
+  assert.throws(() => c.setMaxSendFragment(), invalidArgumentError);
+
+  // Invalid arg is passed
+  [null, undefined, '', {}, false, true].forEach((arg) => {
+    assert.throws(() => c.setMaxSendFragment(arg), invalidArgumentError);
+  });
+
+  [NaN, Infinity, 2 ** 31].forEach((arg) => {
+    assert.throws(() => c.setMaxSendFragment(arg), rangeError);
+  });
+
+  assert.throws(() => c.setMaxSendFragment(new Symbol()), typeError);
```

Comment:
```suggestion
```

Reply:
I don't think this test is useful, I think we should remove it. `Symbol` is not intended to be used with the `new` keyword (see https://tc39.es/ecma262/#sec-symbol-constructor), so `new Symbol` will raise a `TypeError` but that's just JavaScript, it has nothing to do this API.